src/canvas/StaticCanvas.ts

```diff
@@ -1363,16 +1363,16 @@ export class StaticCanvas<
    * @example <caption>Generate dataURL with objects that overlap a specified object</caption>
    * var myObject;
    * var dataURL = canvas.toDataURL({
-   *   filter: (object) => object.isContainedWithinObject(myObject) || object.intersectsWithObject(myObject)
+   *   filter: (object) => object.isOverlapping(myObject)
    * });
    */
-  toDataURL(options = {} as TDataUrlOptions): string {
-    const {
-      format = 'png',
-      quality = 1,
-      multiplier = 1,
-      enableRetinaScaling = false,
-    } = options;
+  toDataURL({
+    format = 'png',
+    quality = 1,
+    multiplier = 1,
+    enableRetinaScaling = false,
+    ...options
+  }: TDataUrlOptions = {}): string {
```

Comment:
to me it doens't make sense to specify a ctx here.
Can we releage this to a specific node item?
If we need to export something in PDF/SVG using node-canvas, shouldn't we add a method that is specific to that and return something related to that?

We don't have code size issues in node and we are not bound to bundling issues.
Having toCanvasElement that return a canvas element that  is good for pdf/svg export buy only if we previously created a specific ctx sounds complicated.

Let's have a toNodePDF and a toNodeSVG ( or whatever it can be called to avoid confusion with current toSVG ) in the node classes.

Take also care that moving away from node-canvas is still a remote option and if there are other libraries that do not require building or that are more canvas compliant, is not granted they will have a pdf or svg export.

Reply:
That is why I didn't expose toPDF etc. So we are not bound to node canvas.

Suggest a different way?